tests/suites/test_suite_mpi.data

```diff
@@ -406,10 +406,10 @@ mbedtls_mpi_shift_r:10:"128":1:10:"64"
 Test mbedtls_mpi_shift_r #2
 mbedtls_mpi_shift_r:10:"120815570979701484704906977000760567182871429114712069861589084706550626575967516787438008593490722779337547394120718248995900363209947025063336882559539208430319216688889117222633155838468458047056355241515415159736436403445579777425189969":45:10:"3433785053053426415343295076376096153094051405637175942660777670498379921354157795219578264137985649407981651226029903483433269093721578004287291678324982297860947730012217028349628999378309630601971640587504883789518896817457"
 
-Test mbedtls_mpi_shift_r #4
+Test mbedtls_mpi_shift_r #4 [#1]
```

Comment:
Numbers of test cases here are wild. 1, 2, 4, 4, 6

Reply:
Rejected as out of scope.